### PR TITLE
Added new icons

### PR DESCRIPTION
**Pull request by oalpatov**
https://sumupteam.atlassian.net/browse/IA-7306

## Purpose

Added new icons:
flip_camera, credit_note, delivery_note, electronic_invoice, recurring_invoice, no_costs

All icons are approved and exists in the [Iconography](https://www.figma.com/file/LnhSTiHtYMVw3Ppw5QZVph/Circuit-UI-Foundation?node-id=11896-67&t=oEFDVn2QjMV1husE-4)

Screenshot 2023-04-27 at 15 06 20
Screenshot 2023-04-27 at 15 06 34
Screenshot 2023-04-27 at 15 06 40

## Approach and changes

_Describe how you solved the problem_

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

**Review by robinmetral — CHANGES_REQUESTED** [review]
Thanks for making sure that this is in the Figma library! There only seems to be a small issue with some icons (those that are vertical—they should have spacing on both sides to be centered in the 24x24 box).

Can you check if you can reexport these icons by selecting the icon's group on Figma? (instead of the vector itself) 👇 

Don't do this:
Screenshot 2023-04-27 at 15 44 46

...do this:
Screenshot 2023-04-27 at 15 44 54

**Comment by robinmetral** [comment]
Otherwise if this doesn't work let me know @oalpatov, and I'll clean them up manually 🙂

**Review by robinmetral — APPROVED** [review]
Looks good to me! Thanks for your contribution 🙌 🎉

**Comment by oalpatov** [comment]
> Looks good to me! Thanks for your contribution 🙌 🎉

Awesome, thanks for reviewing :)

I'm wondering if there is some guide how to publish this changes? I see that I should add something to changlist (but my guess that happening after executing some command)

**Comment by robinmetral** [comment]
> I'm wondering if there is some guide how to publish this changes?

You read my mind, see my reply above 🙂 The documentation is in the CONTRIBUTING.md (good place to start)—if anything is unclear let us know and we'll improve it

**Comment by oalpatov** [comment]
> Ah actually a couple more things:
> 
> * you need to add a changeset so we can release the icons as a new minor for the package. Run `npm run changeset` (or refer to our contribution docs for more details)
> * since you opened the branch from a fork and not on the main repo (do you have access to the `sumup-oss` org? Otherwise I'll send you an invite) the CI is not running. Since this is an icons-only, clearly non-breaking change I'm happy to go ahead an merge it anyways. Next time please open a branch directly in `sumup-oss/circuit-ui` 🙏

I have no access to the `sumup-oss` org (the only reason why I've created a fork). Could you please send me an invite?

File: .changeset/strong-tips-brush.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': minor
+---
+
+Added new icons: flip_camera, credit_note, delivery_note, electronic_invoice, recurring_invoice, no_costs
```
Comment (robinmetral) [inline .changeset/strong-tips-brush.md:5]:
```suggestion
Added new icons in size 24: `FlipCamera`, `CreditNote`, `DeliveryNote`, `ElectronicInvoice`, `RecurringInvoice`, `NoCosts`
```

**Comment by robinmetral** [comment]
@oalpatov invite sent for joining the OSS org 🙂 

I'll handle the release for this. Congrats on your first contribution!

**Pull request merged**